ruby/ql/lib/codeql/ruby/ast/Constant.qll

```diff
@@ -170,6 +170,24 @@ module ConstantValue {
 
   /** A constant `nil` value. */
   class ConstantNilValue extends ConstantValue, TNil { }
+
+  /** Gets the integer constant `x`. */
+  ConstantValue getInt(int x) { result.getInt() = x }
```

Comment:
It feels a bit strange that the "inverse" of `getInt` is named `getInt` . Wouldn't `ConstantValue::fromInt` be a better name?

Reply:
`fromInt` sounds good to me 👍

Reply:
Same of the other getters, of course.

Reply:
Renamed to `fromInt` etc